client/apps/edit/components/section_artworks/index.coffee

```diff
@@ -47,8 +47,19 @@ module.exports = React.createClass
         for r in res._embedded.results
           if r.type == 'Artwork'
             id = r._links.self.href.substr(r._links.self.href.lastIndexOf('/') + 1)
-            vals.push { id: id , value: r.title }
+            vals.push
+              id: id
+              value: r.title
+              thumbnail: r._links.thumbnail.href
         return vals
+      templates:
+        suggestion: (data) ->
+          """
+            <div class='esa-suggestion' \
+                 style='background-image: url(#{data.thumbnail})'>
+            </div>
+            #{data.value}
+          """
```

Comment:
AH! That's what I was trying to do forever. Very cool. Didn't realize it was `templates: suggestion:->`